src/main/java/mate/project/controller/BookController.java

```diff
@@ -27,16 +31,19 @@ public class BookController {
     private final BookService bookService;
 
     @GetMapping
-    public List<BookDto> getAll() {
-        return bookService.findAll();
+    @Operation(summary = "Get all books", description = "Get a list of all available books")
+    public List<BookDto> getAll(Pageable pageable) {
```

Comment:
getAll method has been modified to accept a Pageable parameter but no default Pageable has been set, this could lead to unexpected results if not handled correctly.